update_lmod_cache.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Script to update Lmod cache in EESSI
+#
+
+source utils.sh
```

Comment:
Probably need to ensure we can pick this up from the same directory as the script (so use `$TOPDIR`)